.all-contributorsrc

```diff
@@ -517,6 +517,13 @@
       "contributions": [
         "tool"
       ]
+    },
+    {
+      "login": "binkpitch",
+      "name": "Bink",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/20456165?v=4",
+      "profile": "https://binkpitch.me/",
+      "contributions": []
```

Comment:
🎉 Don't forget to write your contributions here.

Reply:
Thank you for point me out. I will fix this.